repositories/blog.repository.js

```diff
@@ -0,0 +1,110 @@
+const Blog = require("../models/blog.model");
+
+const getAllBlogs = async () => {
+  try {
+    const blogs = await Blog.findAll();
+    return blogs;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+};
+
+const createBlog = async (
+  id,
+  title,
+  description,
+  authorId,
+  createdAt,
+  updatedAt
+) => {
+  try {
+    const blog = await Blog.create({
+      id: id,
+      title: title,
+      description: description,
+      authorId: authorId,
+      createdAt: createdAt,
+      updatedAt: updatedAt,
+    });
+    return blog;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getBlogByBlogId = async (blogId) => {
+  try {
+    const blog = await Blog.findAll({
+      where: {
+        id: blogId,
+      },
+    });
+
+    if (blog) {
+      return blog;
+    }
+    return null;
+  } catch (error) {
+    throw error;
+  }
+};
+
+const getBlogByAuthorId = async (authorId) => {
+  try {
+    const blogs = await Blog.findAll({
+      where: {
+        authorId: authorId,
+      },
+    });
+
+    if (blogs) {
+      return blogs;
+    } 
```

Comment:
what will happen if the authorId doesn't match any data? wouldn't it response an empty array?

Reply:
I send null value in blog service and check if response is null then send data is not found